docs/coverage.md

```diff
@@ -52,15 +51,15 @@
 6. Checks ON-CHAIN REJECTION text to assure rejection is waiting for execution  
 7. Executes Rejection  
 8. Opens history tab, checks last tx is a Rejection and checks the successful status  
-
+  
 #### [Replace owner](./../src/replace_owners.test.js)
 1. Find added owner. click "Replace" button  
 2. Add valid name and address for replacement owner  
 3. Signs and execute  
 4. Removes owner for replacement. Sets threshold to "2". Check owner name and address in review step  
 5. Signs and executes  
 6. Checks status success of executed tx  
-
+  
```

Comment:
🤨

Reply:
I'm using the double space to make line breaks